cmd/release-controller/upgrades.go

```diff
@@ -19,8 +19,9 @@ import (
 )
 
 type UpgradeResult struct {
-	State string `json:"state"`
-	URL   string `json:"url"`
+	State string       `json:"state"`
+	URL   string       `json:"url"`
+	Time  *metav1.Time `json:"time"`
```

Comment:
You should be able to use the transition time for the phase (if we aren’t recording that, I’d prefer to start there)

Reply:
i am using the transition time: https://github.com/openshift/release-controller/pull/133/files#diff-e3c9339bb6ccd7ce43b1c54a3ae88b8fR325

but we weren't tracking/retaining the transition time in the graph.